packages/Organisms/SiteHeader/index.vue

```diff
@@ -68,6 +78,7 @@ import RplMenu from './menu'
 import RplSearch from './search'
 import RplIcon from '@dpc-sdp/ripple-icon'
 import RplLink from '@dpc-sdp/ripple-link'
+import RplButton from '@dpc-sdp/ripple-button'
```

Comment:
Hey @theleebriggs You should able to remove this `RplButton` from here as it didn't get used as I can see.

Reply:
Fixed